docs/tooling/cli-guides/validate-subnets.md

```diff
@@ -0,0 +1,139 @@
+# Be a Subnet validator 
+
+This page will guide you on how to have your node validate a Subnet! Currently, we are only 
+supporting validating on Fuji network with Mainnet support coming in the near future!
+
+:::warning
+
+ALPHA WARNING: This command is currently in experimental mode. Proceed at your own risk.
+
+:::
+
+## Prerequisites
+
+Before we begin, you will need to have:
+
+- Created a Cloud Server node as described [here](/docs/tooling/cli-guides/create-a-validator.md)
+- Node is bootstrapped to Primary Network (run `avalanche node status <clusterName>` to check 
+bootstrap status)
+- Deployed Subnet on Fuji in local machine
+- Stored key / Ledger with AVAX to pay for gas fess associated with adding node as Primary Network 
+and Subnet Validator transactions. Instructions on how to fund stored key on Fuji can be found
+[here](https://docs.avax.network/build/subnet/deploy/fuji-testnet-subnet#funding-the-key)
+
+## Sync with Subnet
+
+Before the node can be a Subnet Validator, the node needs to first sync with the Subnet. 
+
+To sync with a Subnet, run:
```

Comment:
Let's say the node belongs to the cluster myValidatorsCluster, and the subnet name used in CLI is mySubmet. To sync the node to the subnet, run: